STANDARD.md

```diff
@@ -584,19 +590,37 @@ patch order:
 Unit Systems and Dimensionality
 -------------------------------
 
-While this standard does not impose any unit system on the data that is stored
-itself, it still requires a common interface to convert one unit system to
-another.
-
-To allow scaling data without reformatting it during the write process we
-provide a unit conversation factor, often called `unitSI` in the document,
-to transform it to a corresponding quantity in the International System of
-Units (SI).
+All datasets and attributes can be written in arbitrary units, in the openPMD
+file. Typically, this will be the unit system adopted internally by the
+software that writes the openPMD file (e.g. the unit system adopted internally
+by a physics simulation code), in order to avoid a reduction in performance
+associated with unit conversion during the write process.
+
+However, for each dataset and attribute, it is **very strongly** recommended to
+provide the conversion factor from the chosen unit system to the International
+System of Units (SI). (See the description of the attributes `timeUnitSI`, `gridUnitSI` and `unitSI`, in the present document.) This allows
+the reading softwares to convert the datasets to units that any user can
+understand, without requiring them to know the chosen units system.
+
+(Note that, for human readable output, it is possible to add the actual string
```

Comment:
I would not add this note here since it's confusing with out concept of automatically deriving actual units. We might rather add a reference here to `unitDimension` and add advice to implementers and users therein.

Reply:
Ok, I just removed this note altogether.